components/Report.vue

```diff
@@ -336,6 +332,7 @@ export default {
     return {
       units: 'imperial',
       httpErrors: httpErrors,
+      hucPolyExplanation: 'The shaded region on the map is the nearest watershed (hydrological unit, level 12) and is only used to summarize wildfire data around this place.'
```

Comment:
This one was more about reducing duplication because it's sorta fussy language and I expect it to be revised a few times, seemed easier to move it here instead of maybe-missing keeping it in sync (even though it's a close call, imho).  You decide!

Reply:
That makes sense and this does seem like a good way to avoid text duplication. Approved!